STEER/AOD/AliAODTrack.h

```diff
@@ -322,7 +322,7 @@ class AliAODTrack : public AliVTrack {
   Double_t  GetTPCsignalTunedOnData() const { return fTPCsignalTuned;}
   void      SetTPCsignalTunedOnData(Double_t signal) {fTPCsignalTuned = signal;}
   UShort_t  GetTPCsignalN()      const { return fDetPid?fDetPid->GetTPCsignalN():0;    }
-  virtual Bool_t GetTPCdEdxInfo( AliTPCdEdxInfo &v) const {return fDetPid?fDetPid->GetTPCdEdxInfo(v):0;}
+  virtual AliTPCdEdxInfo* GetTPCdEdxInfo() const {return fDetPid?fDetPid->GetTPCdEdxInfo():0;}
```

Comment:
Hello @mikesas, after solving a build problem (triggered by a Clang warning, see alisw/AliRoot#583) I have noticed that... the problem was effectively triggered by this line.

Why have you changed _code_ in a PR mentioning changes only in Doxygen? Is this change needed? Could you please clarify? This change should be reverted, then go in a separate pull request in order to be discussed separately...